src/main/java/viper/gobra/frontend/GobraLexer.java

```diff
@@ -0,0 +1,854 @@
+// Generated from /home/nico/Documents/repositories/projects/eth/BA/gobraHome/gobra/src/main/antlr4/GobraLexer.g4 by ANTLR 4.9.2
```

Comment:
I've tried this plugin before, and it doesn't offer enough options to be useful on its own. It also creates a dependency mess, so it's way easier to just generate the parser manually when changing the grammar and use the generated files otherwise. I agree that we should find a way to make this more automatic, maybe by moving the parser into its own sub-project folder in gobra?

Reply:
What options is this plugin missing to make it usable here? 

Also, what do you mean by dependency mess? Does this plugin assume a different version of sbt or is it something else?

Are there any other alternatives? could we for example configure sbt such that a script is run when we execute `sbt compile`?

If we do not manage to automate it, can we at least configure ANTLR to generate files with valid license headers, and without any reference to a specific user's filesystem?